src/SquareEdit/SelectOptions.ts

```diff
@@ -440,6 +440,30 @@ export function triggerSplitActions (): void {
   document.body.addEventListener('keydown', deleteButtonHandler);
 }
 
+/** 
+ * trigger default actions when selecting by syl
+ */
+export function triggerDefaultSylActions (): void {
+  endOptionsSelection();
+  try {
+    const moreEdit = document.getElementById('moreEdit');
+    moreEdit.classList.remove('is-invisible');
+    moreEdit.innerHTML = Contents.defaultSylActionContents;
```

Comment:
This is fine here but generally we should avoid interacting with `innerHTML` directly since it destroys any events placed on elements contained within.